conf/cmi/user.role.school_editor.yml

```diff
@@ -42,20 +43,23 @@ permissions:
   - 'access user profiles'
   - 'cancel account'
   - 'change own username'
+  - 'create announcement content'
```

Comment:
Add these permissions to `helfi_kasko_content_platform_config_grant_permissions()` hook as well. See https://github.com/City-of-Helsinki/drupal-helfi-platform-config/blob/main/documentation/development.md#drupal-permissions